<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerializeTestClass</class>
 <widget class="QDialog" name="SerializeTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerializeTest</string>
  </property>
  <widget class="QPushButton" name="structBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>struct</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stringBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>string</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DataBufferBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>DataBuffer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SerialPointerBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SerialPointer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vectorBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>vector</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vectorPtrBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>vector+Ptr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InterfaceBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Interface</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SerialObjectBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SerialObject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExceptionBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exception</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="serializetest.qrc"/>
 </resources>
 <connections/>
</ui>
